spec/features/posts/show_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'posts#show', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Lilly', photo: 'lilly.jpg', bio: 'Software Developer', posts_counter: 0)
+    @post = Post.create(author: @user, title: 'Post 1', text: 'This is Post 1')
+
+    5.times { |i| Comment.create(post: @post, user: @user, text: "Comment #{i + 1} for Post 1") }
+
+    visit user_post_path(@user, @post)
+  end
+
+  it 'displays the post details' do
+    expect(page).to have_content(@post.title)
+    expect(page).to have_content(@post.text)
+    expect(page).to have_content('Comments: 5')
+    expect(page).to have_content('Likes: 0')
+  end
+
+  it 'displays comments for the post' do
+    @post.comments.each do |comment|
+      expect(page).to have_content(comment.user.name)
+      expect(page).to have_content(comment.text)
+    end
```

Comment:
Nice job so far👍🏼. you seem to have missed some tests here😉. Just as [required](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md), kindly add the following tests👍🏼.

- I can see who wrote the post.
<img width="409" alt="image" src="https://github.com/Innocent259/blog-app/assets/64643995/32c10405-6a36-4a16-9f8e-af73eaa25239">